content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -431,45 +480,44 @@ Some values of an object are typically generated before the object is persisted.
 
-Resource versions are strings that identify the server's internal version of an object. Resource versions can be used by clients to determine when objects have changed, or to express data consistency requirements when getting, listing and watching resources. Resource versions must be treated as opaque by clients and passed unmodified back to the server. For example, clients must not assume resource versions are numeric, and may only compare two resource versions for equality (i.e. must not compare resource versions for greater-than or less-than relationships).
+## Resource versions
 
-### ResourceVersion in metadata
+Resource versions are strings that identify the server's internal version of an object. Resource versions can be used by clients to determine when objects have changed, or to express data consistency requirements when getting, listing and watching resources. Resource versions must be treated as opaque by clients and passed unmodified back to the server. For example, clients must not assume resource versions are numeric, and may only compare two resource versions for equality (this means that you must not compare resource versions for greater-than or less-than relationships).
+
+### `ResourceVersion` in metadata {#resourceversion-in-metadata}
 
 Clients find resource versions in resources, including the resources in watch events, and list responses returned from the server:
 
 [v1.meta/ObjectMeta](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#objectmeta-v1-meta) - The `metadata.resourceVersion` of a resource instance identifies the resource version the instance was last modified at.
 
 [v1.meta/ListMeta](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#listmeta-v1-meta) - The `metadata.resourceVersion` of a resource collection (i.e. a list response) identifies the resource version at which the list response was constructed.
 
-### The ResourceVersion Parameter
+### `ResourceVersion` parameter {#the-resourceversion-parameter}
 
-The get, list, and watch operations support the `resourceVersion` parameter.
+The **get**, **list**, and **watch** operations support the `resourceVersion` parameter.
 
 The exact meaning of this parameter differs depending on the operation and the value of `resourceVersion`.
 
 For get and list, the semantics of resource version are:
 
-**Get:**
+**get:**
 
 | resourceVersion unset | resourceVersion="0" | resourceVersion="{value other than 0}" |
 |-----------------------|---------------------|----------------------------------------|
 | Most Recent           | Any                 | Not older than                         |
 
-**List:**
+**list:**
 
-v1.19+ API servers support the `resourceVersionMatch` parameter, which
-determines how resourceVersion is applied to list calls.  It is highly
+From version v1.19, Kubernetes API servers support the `resourceVersionMatch` parameter,
+which determines how resourceVersion is applied to list calls.  It is highly
```

Comment:
nit: ... how the `resourceVersion` parameter is applied to list calls?

Reply:
Addressed, I think.